salttesting/jenkins.py

```diff
@@ -400,32 +426,49 @@ def bootstrap_cloud_minion(options):
     if options.insecure:
         script_args.append('-I')
     if options.bootstrap_salt_url != SALT_GIT_URL:
-        script_args.extend([
-            '-g', options.bootstrap_salt_url
-        ])
-    script_args.extend(['git', options.bootstrap_salt_commit])
+        script_args.extend(['-g', options.bootstrap_salt_url])
+    if options.bootstrap_salt_commit:
+        script_args.extend(['git', options.bootstrap_salt_commit])
 
     cmd = [
         'salt-cloud',
         '-l', options.log_level,
-        '--script-args="{0}"'.format(' '.join(script_args)),
         '-p', options.vm_source,
         '--out=yaml',
         '--no-color',
         options.vm_name
     ]
+    if getattr(options, 'windows', False):
+        # This switch is needed until we fix the problem with salt-cloud
+        # deleting the installation file on CentOS 7
```

Comment:
For some reason, Salt-Cloud can't delete the Salt Installation file after it's finished installing. I only saw this problem on the Cent7 box I was working on. It wasn't a problem on my Ubuntu master. Since the whole VM gets blasted anyway at the end of the tests, I didn't see a problem leaving the file on the machine.